daemons/fenced/fenced_remote.c

```diff
@@ -2216,7 +2243,7 @@ fenced_process_fencing_reply(xmlNode *msg)
         } else {
             op->state = st_failed;
         }
-        remote_op_done(op, msg, pcmk_rc2legacy(stonith__result2rc(&result)), FALSE);
+        finalize_op(op, msg, &result, false);
```

Comment:
one of these cases where we are assuming that result doesn't have reason set to something dynamic owned by result and thus doesn't need to be reset - right? if yes can we ... or better should we thinking of possible future changes?

Reply:
reason is always dynamically allocated, it simplifies things considerably

This is the same function as before in need of a reset (anywhere it returns)